node/resolvers/paths.ts

```diff
@@ -95,6 +95,10 @@ const paths = {
     filterUser: (email, customFields?) => join(',', [`http://api.vtex.com/${account}/dataentities/CL/search?email=${email}&_fields=userId,id,firstName,lastName,birthDate,gender,homePhone,businessPhone,document,email,tradeName,corporateName,stateRegistration,corporateDocument`, customFields]),
     profile: (id) => `http://api.vtex.com/${account}/dataentities/CL/documents/${id}`,
   }),
+
+  logistics: account => ({
```

Comment:
I think is better
`logisticsConfig: account => ({http://${account}.vtexcommercestable.com.br/api/logistics/pub/shipping/configuration})`